src/Annotator.js

```diff
@@ -101,7 +102,19 @@ class Annotator extends EventEmitter {
         // Get the container dom element if selector was passed, in tests
         this.container = this.options.container;
         if (typeof this.options.container === 'string') {
-            this.container = document.querySelector(this.options.container);
+            this.container = document.querySelector(this.container);
+        }
+
+        // Get the header dom element if selector was passed, in tests
+        this.headerElement = this.options.headerElement;
+        if (typeof this.headerElement === 'string') {
+            this.headerElement = document.querySelector(this.headerElement);
+        }
+
+        // If using box content preview header and no external header element was specified,
+        // fallback to the container element
+        if (this.options.header !== 'none' && !this.headerElement) {
+            this.headerElement = this.container.querySelector(SELECTOR_BOX_PREVIEW_HEADER_CONTAINER);
```

Comment:
@ConradJChan It seems like this class is never loaded onto the preview header? The default preview header seems to use `bp-header`